README.md

```diff
@@ -1,3 +1,26 @@
 # Agola Site
 
-This is the repository for the [agola site](https://agola.io)
\ No newline at end of file
+This is the repository for the [agola site](https://agola.io)
+
+
+## Contribute
+
+In order to check the web site on your local computer before you send your contribution you'll need to
+
+install [nodejs](https://nodejs.org/en/)
+
+Then you need to install [vuepress](https://vuepress.vuejs.org/) and all the other dependencies using:
+
+```
+npm install
+```
+
+and run the web server locally:
+
+```
+cd site
+../node_modules/vuepress/cli.js dev
```

Comment:
The right command is 

```
vuepress dev site
```

Reply:
sorry I considered using a globally installed vuepress. To use the locally installed vuepress perhaps it'll be better to just add a npm command like

```
"dev": "vuepress dev site"
```

in package.json

and then suggest to use

`npm run dev`